### PR TITLE
fix: s1 download file

### PR DESCRIPTION
**Pull request by davidt99**

### CODE REVIEW

File: edr_connect/proxies/s1_edr_proxy.py [hunk 1]
```diff
@@ -140,17 +148,21 @@ def assert_response(self, response: requests.Response):
...
+    def _fetch_file(self,
+                    alert_info: AlertInfo,
+                    fetched_alerts: Dict[str, S1FetchCommand]) -> Tuple[Optional[str], Optional[str]]:
         zip_password = secrets.token_urlsafe(32)
-        fetch_file_time = datetime.datetime.utcnow() - datetime.timedelta(seconds=5)
+        alert_id = alert_info.alert_id
         fetched_alert = fetched_alerts.get(alert_id)
         if not fetched_alert:
-            _logger.debug('Sending fetch command to the endpoint')
+            fetch_file_time = datetime.datetime.utcnow() - datetime.timedelta(seconds=5)
```
Comment (yoniabrahamy) [inline edr_connect/proxies/s1_edr_proxy.py:158]: Why?

File: edr_connect/proxies/s1_edr_proxy.py [hunk 2]
```diff
@@ -46,6 +46,14 @@ class NoteInfo:
     text: str
 
 
+@dataclass
+class S1FetchCommand(FetchCommand):
+    fetch_time: datetime.datetime
+
+    def __hash__(self):
```
Comment (yoniabrahamy) [inline edr_connect/proxies/s1_edr_proxy.py:53]: Why not have this in the base class as well?

**Review by yoniabrahamy — APPROVED** [review]

File: edr_connect/proxies/s1_edr_proxy.py (hunk 2 above)
Comment (davidt99) [reply edr_connect/proxies/s1_edr_proxy.py:53]: It doesn't get it from the parent

File: edr_connect/proxies/s1_edr_proxy.py (hunk 1 above)
Comment (davidt99) [reply edr_connect/proxies/s1_edr_proxy.py:158]: we talked

**Pull request merged**